src/pages/preferences/fieldMaps.ts

```diff
@@ -0,0 +1,173 @@
+export const timeZones = [
```

Comment:
Created https://jira.mongodb.org/browse/EVG-7967 to track not having to do this

Reply:
Cool i assumed these were validated by the server somehow